hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/events/HmppsDomainEventsEmitterIntTest.kt

```diff
@@ -549,9 +550,10 @@ class HmppsDomainEventsEmitterIntTest : IntegrationTestBase() {
     await untilCallTo { prisonApi.countFor("/api/prisoner-search/offenders/$prisonerNumber") } matches { it == 1 }
 
     // delete create events
-    await untilCallTo { getNumberOfMessagesCurrentlyOnDomainQueue() } matches { it != 0 }
+    await atMost Duration.ofSeconds(30) untilCallTo { getNumberOfMessagesCurrentlyOnDomainQueue() } matches { it != 0 }
 
     await untilCallTo { prisonerHashRepository.findById(prisonerNumber) } matches { it != null }
+    await untilCallTo { prisonerRepository.getSummary(prisonerNumber, SyncIndex.RED) } matches { it != null }
```

Comment:
This was important, to wait till the prisoner is re-created in the red index too